### ISSUE TITLE
Unable to download book data.zip

### ISSUE THREAD

**Issue by GrahamOakford** [fzumstein/python-for-excel#14]
Unable to open python-for-excel-1st-edition.zip on windows 11.  Zip file downloaded OK to Downloads folder, however, attempts to extract, open are futile.  Windows dialog "Compressed (zipped) Folders Error" reports that the C:\Users\...\Downloads\python-for-excel-1st-edition.zip is invalid.

**Comment by GrahamOakford** [comment]
Actually, I have given this an inappropriate name. I can download the file but Winzip will not open it and says it is an invalid archive file

**Comment by fzumstein** [comment]
Can you right-click and select `Unzip all...` (the native Windows function)? I've just tried and had no issues.

**Comment by GrahamOakford** [comment]
That is how I get the invalid zip archive message. I have tried two
different machines and rebooted between attempts.

I plan to try on my linux machine after work. I am wondering if its a
Windows issue. Or possibly a security issue at work.
I can let you know if I have success or if not, what I get from linux at
home.

thank you for responding

kind regards

Graham Oakford.

**Comment by fzumstein** [comment]
The ZIP contains macro-enabled workbooks, maybe they get flagged by an anti-virus?

**Comment by GrahamOakford** [comment]
Well Felix, issue resolved!

The file I downloaded on my work laptop today was 4.3Mb

[image: image.png]

Whereas the file I downloaded to my Ubuntu Linux box this evening is
17.7Mb.

[image: Screenshot from 2023-11-23 20-34-00.png]

For some reason my work has a security policy that does not allow emailing
or downloading of python files. In addition to incomplete downloads.
I will not make any derogatory comments about them, I have the files and I
prefer to use linux anyway.

The issue is I do write python automation for regulatory reporting for
EnergyAustralia and I purchased your book to see if I can learn anything
new. I will read and look through your examples thanks for writing it and
thanks for responding to my issue on Github.

Kind regards

*Graham Oakford*
